ru.linux_gaming.PortProton.yml

```diff
@@ -26,6 +26,7 @@ finish-args:
   - --system-talk-name=org.freedesktop.NetworkManager
   - --filesystem=xdg-run/gamescope-0
   - --filesystem=xdg-run/pipewire-0
+  - --filesystem=xdg-config/MangoHud
```

Comment:
Why? Following the instructions you can just create the config file in `~/.var`.

This is not allowed by default. You'd need an exception for this https://docs.flathub.org/docs/for-app-authors/linter#finish-args-unnecessary-xdg-config-access

Reply:
> Why? Following the instructions you can just create the config file in `~/.var`.
> 
> This is not allowed by default. You'd need an exception for this https://docs.flathub.org/docs/for-app-authors/linter#finish-args-unnecessary-xdg-config-access

I thought it was more convenient to take configs from the system, and I also saw it in Lutris, but there in the manifest is written .config/MangoHud, but if you can not do that, then okay, I'll remove it then